nethttp/server_test.go

```diff
@@ -273,3 +273,59 @@ func BenchmarkStatusCodeTrackingOverhead(b *testing.B) {
 		}
 	})
 }
+
+func TestMiddlewareHandlerPanic(t *testing.T) {
+	recovery := func(f http.HandlerFunc) http.HandlerFunc {
+		return func(w http.ResponseWriter, r *http.Request) {
+			defer func() {
+				if err := recover(); err != nil {
+					t.Logf("[Recovery] panic recovered\n")
+					w.WriteHeader(http.StatusInternalServerError)
+				}
+			}()
+			f(w, r)
+		}
+	}
+	tests := []struct {
+		handler         func(w http.ResponseWriter, r *http.Request)
+		spanFinishCount int
+		tag             string
```

Comment:
Is 'tag' used?

Reply:
@stuart-mclaren 
https://github.com/YueHonghui/go-stdlib/blob/c11e8f1250e92438c9e2c7985966cc9d468fe944/nethttp/server_test.go#L312

Yes, `tag` is used here as part of TestCase name.